### PR TITLE
Add contribution quick start, repeat acquisition, configuration and livestream

### PR DESCRIPTION
**Pull request by dgmccart**
Adding docs contribution quick start guide, tutorial on repeated acquisitions, and livestreaming tutorial that links to an example python file for napari livestreaming

### CODE REVIEW

**Comment by aliddell** [comment]
FWIW when I try to check out this PR I get

```
error: invalid path 'docs/contribution/for_contributors.md.'
failed to run git: exit status 1
```

File: docs/tutorials/storage.md [hunk 1]
```diff
@@ -60,4 +60,4 @@ config.video[0].storage.identifier = manager.select( acquire.DeviceKind.Storage,
 config.video[0].storage.settings.filename = "out.tif" 
 ```
 
-Before proceeding, complete the `Camera` setup and call `runtime.set_configuration(config)` to save those new configuration settings.
\ No newline at end of file
+Before proceeding, complete the `Camera` setup and call `set_configuration` to save those new configuration settings.
```
Comment (aliddell) [inline docs/tutorials/storage.md:63]: Why this change?

File: docs/tutorials/livestream.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Livestream to napari
+
+The below script can be used to livestream data to the [napari viewer](https://napari.org/stable/). You may also utilize the `Acquire` napari plugin, which is provided in the package upon install. You can access the plugin in the napari plugins menu once `Acquire` is installed. You can review the [plugin code here](https://github.com/acquire-project/acquire-python/blob/main/python/acquire/__init__.py). You may also stream using other packages such at `matplotlib`.
```
Comment (aliddell) [inline docs/tutorials/livestream.md:3]:
```suggestion
The below script can be used to livestream data to the [napari viewer](https://napari.org/stable/). You may also utilize the `Acquire` napari plugin, which is provided in the package upon install. You can access the plugin in the napari plugins menu once `Acquire` is installed. You can review the [plugin code here](https://github.com/acquire-project/acquire-python/blob/main/python/acquire/__init__.py#L131). You may also stream using other packages such at `matplotlib`.
```

File: docs/tutorials/storage.md (hunk 1 above)
Comment (dgmccart) [reply docs/tutorials/storage.md:63]: I wanted to keep the tutorials short. I can add a reference to another tutorial that describes how to setup the camera.

Comment (dgmccart) [reply docs/tutorials/storage.md:63]: To match formatting in other tutorials

**Review by aliddell — APPROVED** [review]
